examples/checkout-book.yaml

```diff
@@ -33,8 +33,8 @@
     provider: https://library.example/
     receiver: https://bob.example/
     resourceQuantity: 
-      qudt:unit: unit:Number
-      qudt:numericValue: 1
+      om2:hasUnit: om2:one
```

Comment:
@jvanbockryck don't worry too much about all of the om2:one comments, Bob was helpfully flagging them in case we do want to try to do something with defining our own unit for "each".  He's going to do some more research.

Reply:
The "om:one" is equivalent of "unit:Number" with only one dimension.